src/sentry/notifications/utils/participants.py

```diff
@@ -159,7 +160,7 @@ def get_owners(
     project: Project,
     event: Event | None = None,
     fallthrough_choice: FallthroughChoiceType | None = None,
-) -> Sequence[Team | APIUser]:
+) -> List[Team | APIUser]:
```

Comment:
Change return type to a `List` so we can append suspect committers to it [here](https://github.com/getsentry/sentry/pull/44361/files#diff-5996232f4fd7afb156ac89ae6b48c03b48102576ca43bfe45e55ec4da422acaeR308)